.github/workflows/ci.yml

```
@@ -1,6 +1,12 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  pull_request:
+    branches:
+      - dev
+  push:
+    branches: 
+      - dev
```

Let's add master, too. We have slightly different gradle options for the master branch
```suggestion
      - dev
      - master
```

---

we also had the suggestion to run this on any branch we push against, does this make sense?